openedx_external_enrollments/tests/external_enrollments/test_greenfig_external_enrollment.py

```diff
@@ -0,0 +1,77 @@
+"""Tests EdxInstanceExternalEnrollment class file."""
+from django.test import TestCase
+from mock import Mock, patch
+
+from openedx_external_enrollments.external_enrollments.greenfig_external_enrollment import (
+    GreenfigInstanceExternalEnrollment,
+)
+
+
+class GreenfigInstanceExternalEnrollmentTest(TestCase):
+    """Test class for GreenfigInstanceExternalEnrollment."""
+
+    @patch('openedx_external_enrollments.external_enrollments.greenfig_external_enrollment.configuration_helpers')
+    def setUp(self, configuration_helpers_mock):  # pylint: disable=arguments-differ
+        """setUp."""
+        configuration_helpers_mock.get_value.return_value = 'setting_value'
+        self.base = GreenfigInstanceExternalEnrollment()
+
+    @patch('openedx_external_enrollments.external_enrollments.greenfig_external_enrollment.get_user')
+    @patch('openedx_external_enrollments.external_enrollments.greenfig_external_enrollment.GreenfigInstanceExternalEnrollment._get_course_list')  # noqa pylint: disable=line-too-long
+    @patch('openedx_external_enrollments.external_enrollments.greenfig_external_enrollment.datetime')
+    def test_get_enrollment_data_enroll(self, datetime_now_mock, _get_course_list_mock, get_user_mock):
```

Comment:
test_get_enrollment_data_**enroll**()? you mean: test_get_enrollment_data()

Reply:
done